pkg/glusterblock/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 The Gluster CSI Authors.
```

Comment:
can we use single dockerfile for both, i feel a lot of code duplication here, can we make use of arguments during docker build?